src/recipes/lint.ts

```diff
@@ -14,6 +14,41 @@ const existsEslintConfiguration = (toolbox: CycliToolbox): boolean =>
       )
   )
 
+const generateConfigForBareReactNative = async (
+  toolbox: CycliToolbox,
+  context: ProjectContext,
+  withPrettier: boolean
+): Promise<void> => {
+  await toolbox.dependencies.addDev('@react-native/eslint-config', context)
+  await toolbox.dependencies.addDev('eslint-plugin-ft-flow', context)
+
+  await toolbox.template.generate({
+    template: join('lint', '.eslintrc.json.ejs'),
+    target: '.eslintrc.json',
```

Comment:
Shouldn't you check if it's not already created?

Reply:
We check it before

```typescript
  if (!existsEslintConfiguration(toolbox)) {
    const generateExpoSpecificConfig =
      isEslintConfigExpoInstalled || toolbox.projectConfig.isExpo()

    if (generateExpoSpecificConfig) {
      await generateConfigForExpo(toolbox, context)
    } else {
      await generateConfigForBareReactNative(toolbox, context, withPrettier)
    }
  }

```